### ISSUE TITLE
Add Support for Conda Package

### ISSUE THREAD

**Issue by RoyiAvital** [MiXaiLL76/faster_coco_eval#50]
It would be great to be able to install the project using `conda`.

Any chance adding a distribution through `conda-forge`?

**Comment by MiXaiLL76** [comment]
Hi! At the moment, I and the team I work in do not use conda.
In general, you can install a package from pip in conda, there are not many dependencies in the project.

But I will study the possibility of publishing the project in the conda repository.

**Comment by RoyiAvital** [comment]
That would be great.
I think for a low dependency project it should be simple. Though have no experience with that.

**Comment by MiXaiLL76** [comment]
https://github.com/conda-forge/staged-recipes/pull/28684

**Comment by RoyiAvital** [comment]
You're quick. Much appreciated.

**Comment by MiXaiLL76** [comment]
> You're quick. Much appreciated.

https://anaconda.org/conda-forge/faster-coco-eval

### Conda install

```bash
conda install conda-forge::faster-coco-eval
```

**Comment by RoyiAvital** [comment]
@MiXaiLL76 , I work with it on Windows which works great.
I tried to install is on MacOS (ARM 64) yet it won't install.

Could you add support for MacOS on Apple Silicon?

**Comment by MiXaiLL76** [comment]
> [@MiXaiLL76](https://github.com/MiXaiLL76) , I work with it on Windows which works great. I tried to install is on MacOS (ARM 64) yet it won't install.
> 
> Could you add support for MacOS on Apple Silicon?

Are you trying to install a conda package?

**Comment by RoyiAvital** [comment]
Yes, I always use the `conda` package. Hence I wrote it on this issue.

**Comment by MiXaiLL76** [comment]
> Yes, I always use the `conda` package. Hence I wrote it on this issue.

I'll check the conda builds for macos.
There might be some errors there.

**Comment by MiXaiLL76** [comment]
> Yes, I always use the `conda` package. Hence I wrote it on this issue.

There is a release 1.6.6 in conda, can you check if it works for you?

**Comment by RoyiAvital** [comment]
Hi, I will try to get it and report.
